app/page.js

```diff
@@ -1,17 +1,21 @@
-import Image from 'next/image'
-import style from './page.module.css'
-import Link from 'next/link'
-import Temperatura from './temperatura'
+import Image from "next/image";
+import style from "./page.module.css";
+import Link from "next/link";
+import Temperatura from "./temperatura";
+import logo from "../public/github-mark-white.png"
 
 export default function Page() {
-    return (
-        <div className={style.main}>
-            <Image className={style.img} src="/avatar.jpg" width={250} height={250} />
-            <h1 className={style.card}>Gilgorri Santiago</h1>
-            <h2 className={style.card}>Santiagogilgorri@gmail.com</h2>
-            <Link href="https://github.com/SG-SantiagoGilgorri" target="_blank"><h2 className={style.card}>https://github.com/SG-SantiagoGilgorri</h2></Link>
-            <Temperatura />
-            {/*<Link href="/home"><button>HomePage</button></Link>*/}
-        </div>
-    )
+  return (
+    <div className={style.div}>
+      <title>MardelClima</title>
+      <Image className={style.img} src="/avatar.jpg" width={205} height={205} />
+      <h1 className={style.card}>Gilgorri Santiago</h1>
+      <h2 className={style.card}>Santiagogilgorri@gmail.com</h2>
```

Comment:
La mayúscula puede traer problemas para el email!
```suggestion
      <h2 className={style.card}>santiagogilgorri@gmail.com</h2>
```